sys/oom_linux_test.go

```diff
@@ -55,6 +55,9 @@ func TestSetNegativeOomScoreAdjustmentWhenUnprivilegedHasNoEffect(t *testing.T)
 
 	initial, adjustment, err := adjustOom(-123)
 	assert.NoError(t, err)
+	if initial < 0 {
```

Comment:
Would you please add some references about why initial < 0 means unpriviledged user? thanks

Reply:
Not means unpriviledged user. Just unpriviledged user can change oom_score_adj if the origin value is negative. I haven't searched the kernel code, but at least it behaviors like that on my system.

Reply:
oh, I think the case here is unpriviledged user can increase oom_score_adj. let me fix that

Reply:
Indeed, on https://docs.kernel.org/filesystems/proc.html#proc-pid-oom-adj-proc-pid-oom-score-adj-adjust-the-oom-killer-score

It says

> The value of /proc/<pid>/oom_score_adj may be reduced no lower than the last value set by a CAP_SYS_RESOURCE process. To reduce the value any lower requires CAP_SYS_RESOURCE.

Reply:
ping @AkihiroSuda

Reply:
Test is changed accordingly, PTAL.